pkg/deploy/assets/cluster-development-predeploy.json

```diff
@@ -104,6 +104,7 @@
                 },
                 "accessPolicies": [],
                 "enabledForDiskEncryption": true,
+                "enableRbacAuthorization": false,
```

Comment:
That's for a different storage account. There are three key vaults that get deployed in dev: the MSI one and two others. The other two use the legacy key vault access policies instead of Entra ID RBAC, which is why I made this attribute configurable through the `keyVault()` function and only set it to true for the MSI storage account.